plugins/webhooks/index.js

```diff
@@ -310,6 +311,93 @@ function pullRequestEvent(pluginOptions, request, reply, parsed) {
         .catch(err => reply(boom.wrap(err)));
 }
 
+/**
+ * Check the pipeline have triggered job or not
+ * @method  hasTriggeredJob
+ * @param   {Pipeline}  pipeline    The pipeline to check
+ * @param   {String}    startFrom   The trigger name
+ * @returns {boolean}               If true the pipeline has triggered job
+ */
+function hasTriggeredJob(pipeline, startFrom) {
+    const nextJobs = workflowParser.getNextJobs(pipeline.workflowGraph, {
+        trigger: startFrom
+    });
+
+    return nextJobs.length > 0;
+}
+
+/**
+ * Get all pipelines which has triggered job
+ * @method  triggerPipelines
+ * @param   {pipelineFactory}   pipelineFactory the pipeline factory to get branch list
```

Comment:
Capitalize the JSDoc descriptions (`The pipeline factory...`)

Reply:
Oops I will fix.